Skills.md

```diff
@@ -13,6 +13,7 @@ This API currently supports versions 4.1 and 4.2. Version is passed in the Accep
 | language            | No | A string determining the language (total 22 languages supported) in which the input text is written. Default value is en. Note that the input parameter passing to language should be the language id. |
 | threshold      | No | A double value between 0 and 1 controlling minimum relevancy scores for skill recognition. Higher values will more tightly restrict the returned skill tags. Default is 0.5. A threshold of 0 means all seed skill phrases recognized by exact string matching will be returned. Note that this parameter is only supported for inputs in English. |
 | auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en,fr, and de. |
+| return_relevant_skills | No | A boolean value indicating whether to return relevant skills for each extracted skill. Defaults to false. Relevant skills are identical to extracted skills, with the confidence rating their relevance to the extracted skill. |

```

Comment:
I think the # of relevant skills may vary differently on skills, some may simply have more siblings while others don't. Instead of using a max # of relevant skills, I would like to suggest a base threshold (all relevant skills higher than that will be returned), but the problem is how to determine that, whether we fix it or ask user to supply parameters.